lib/components/narrative/line-itin/itin-summary.tsx

```diff
@@ -103,8 +102,7 @@ export class ItinerarySummary extends Component<Props> {
     const minTotalFare = minTNCFare * 100 + transitFare
     const maxTotalFare = maxTNCFare * 100 + transitFare
 
-    const startTime = itinerary.startTime + (timeOptions?.offset || 0)
```

Comment:
I didn't find other places that refer to `timeOptions`, did you have any in mind?

Reply:
Sorry I should've been more clear -- I was referring to `offset`

Reply:
In the past, `timeOptions.offset` would contain any time offsets resulting from daylight savings transitions, so that components render the correct time. react-intl/formatjs already handle rendering DST shifts, so we dropped timeOptions.offset when we added react-intl.